README.md

```diff
@@ -66,6 +75,13 @@ Directions.shared.calculate(options) { (waypoints, routes, error) in
 }
 ```
 
+#### Required Info.plist Keys
+Mapbox Navigation requires a few additions to your `Info.plist`. Be sure to sign up or log in to your Mapbox account and grab a [Mapbox Access Token](https://www.mapbox.com/studio/account/tokens/).
+
+1. Add a `MGLMapboxAccessToken` key and paste your [Mapbox Access Token](https://www.mapbox.com/studio/account/tokens/)
+1. Add a `NSLocationWhenInUseUsageDescription` key if you haven't already
```

Comment:
Ah hrmm, yes, tricky. Do we encourage developers to add this to their Info.plist themselves or use the `Capabilities` tab -> `Background Modes`?